docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Integral numeric types - C# reference"
+description: "Learn the range, storage size, and uses for each of the integral numeric types."
+ms.date: 06/25/2019
+f1_keywords:
+  - "byte"
+  - "byte_CSharpKeyword"
+  - "sbyte_CSharpKeyword"
+  - "sbyte"
+  - "short"
+  - "short_CSharpKeyword"
+  - "ushort"
+  - "ushort_CSharpKeyword"
+  - "int_CSharpKeyword"
+  - "int"
+  - "uint"
+  - "uint_CSharpKeyword"
+  - "long_CSharpKeyword"
+  - "long"
+  - "ulong_CSharpKeyword"
+  - "ulong"
+helpviewer_keywords: 
+  - "integral types, C#"
+  - "Visual C#, integral types"
+  - "types [C#], integral types"
+  - "ranges of integral types [C#]"
+  - "byte keyword [C#]"
+  - "sbyte keyword [C#]"
+  - "short keyword [C#]"
+  - "ushort keyword [C#]"
+  - "int keyword [C#]"
+  - "uint keyword [C#]"
+  - "long keyword [C#]"
+  - "ulong keyword [C#]"
+---
+# Integral numeric types  (C# reference)
+
+The **integral types** are a subset of the **numeric types**. The **numeric types** are a subset of the **simple types**. All integral types are value types and can be initialized with *literals*, for example "24". All integral types support [arithmetic](../operators/arithmetic-operators.md), [bitwise logical](../operators/bitwise-and-shift-operators.md), [comparison, and [equality](../operators/equality-operators.md) operators.
+
+## Sizes and ranges
+
+The following table shows the sizes and ranges of the integral types:
+
+|Type|Range|Size|  
+|----------|-----------|----------|  
+|`sbyte`|-128 to 127|Signed 8-bit integer|  
+|`byte`|0 to 255|Unsigned 8-bit integer|  
+|`short`|-32,768 to 32,767|Signed 16-bit integer|  
+|`ushort`|0 to 65,535|Unsigned 16-bit integer|  
+|`int`|-2,147,483,648 to 2,147,483,647|Signed 32-bit integer|  
+|`uint`|0 to 4,294,967,295|Unsigned 32-bit integer|  
+|`long`|-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807|Signed 64-bit integer|  
+|`ulong`|0 to 18,446,744,073,709,551,615|Unsigned 64-bit integer|  
+
+The default value for all integral types is `0`. Each of the integral types has constants named `MinValue` and `MaxValue` for the minimum and maximum value for that type.
+
```

Comment:
The paragraph about `BigInteger` (line 107) might fit more in this section about sizes and ranges as the last paragraph.